vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/ad_notifications/ad_notifications_frequency_capping.cc

```diff
@@ -58,6 +59,11 @@ bool FrequencyCapping::IsAdAllowed() {
     return false;
   }
 
+  FullScreenModeFrequencyCap full_screen_node_frequency_cap;
```

Comment:
nit: should be `full_screen_mode_frequency_cap`? ("m" instead of "n")

Reply:
Fixed